test/e2e/bpmn.navigation.test.ts

```diff
@@ -55,4 +82,15 @@ describe('diagram navigation', () => {
     const image = await page.screenshot({ fullPage: true });
     expect(image).toMatchImageSnapshot(imageSnapshotConfigurator.getConfig(fileName));
   });
+
+  it.each(['zoom in', 'zoom out'])(`ctrl + mouse: %s`, async (zoom: string) => {
+    const deltaX = zoom === 'zoom in' ? -100 : 100;
+    // simulate mouse+ctrl zoom
+    await page.mouse.move(viewportCenterX, viewportCenterY);
+    await page.keyboard.down('Control');
```

Comment:
I forgot to check it but it seems it is working well on Mac in our GitHub 'checks' :)

Reply:
puppeteer is running Chromium. Let's review this in #773